go.mod

```diff
@@ -4,23 +4,24 @@ go 1.17
 
 require (
 	github.com/buger/jsonparser v1.1.1
-	github.com/c-bata/go-prompt v0.2.5
+	github.com/c-bata/go-prompt v0.2.5 // Should not be updated to 0.2.6
```

Comment:
Let's add to this comment an explanation as to why this dependency should not be updated.